cypress/integration/1230-doi_minting/doi/doi.js

```diff
@@ -0,0 +1,253 @@
+  cy.get('[data-testid=menu-button]').click();
+  cy.get('[data-tesid=log-out-link]').click();
+  cy.login(USER_CURATOR_DRAFT_DOI);
+  cy.get('[data-testid=menu-button]').click();
+  cy.get('[data-tesid=worklist-link]').click();
+  cy.get('[data-testid^=message-title]').filter(`:contains(${draftRegistrationPublishWithRequestedDoi})`);
+});
+
+//   @358
+//   Scenario: Curator opens a Registration from a DOI Request Worklist Item
+Given('that a Curator views details of a Worklist item', () => {
+  cy.login(USER_CURATOR_DRAFT_DOI);
+  cy.get('[data-testid="menu-button"]').click();
+  cy.get('[data-testid="worklist-link"]').click();
+  cy.get('[data-testid^=message-type]').last().click();
+});
+And('the item is a DOI request', () => {
+  cy.get('[data-testid^=message-type]').last().contains('DOI Request');
+});
+When('they click "Go to Registration"', () => {
+  cy.get('[data-testid^=go-to-registration]').last().click();
+});
+Then('they see the Registration is opened in the Wizard', () => {
+  cy.get('[data-testid=public-registration-status]').should('be.visible');
+});
+And('they see the Submission tab', () => {});
+And('they see the Create DOI button is enabled', () => {});
+And('they see the Decline DOI button is enabled', () => {});
+
+//   @512
+//   Scenario: A Curator approves a DOI request
+Given('that a Curator opens a Registration from a DOI Request Worklist Item', () => {
+  cy.login(USER_CURATOR_DRAFT_DOI);
+  cy.get('[data-testid="menu-button"]').click();
+  cy.get('[data-testid="worklist-link"]').click();
+  cy.get('[data-testid^=message-title]').filter(`:contains('Published registration with DOI')`).click();
+  cy.get('[data-testid^=message-title]')
+    .filter(`:contains('Published registration with DOI)`)
+    .parent()
+    .within(() => {
+      cy.get('[data-testid^=go-to-registration]').click();
+    });
+});
+When('they click Create DOI', () => {
+  cy.get('[data-testid^=button-toggle-]').first().click();
+  cy.get('[data-testid=button-send-doi-request]');
+});
+Then('they see the Landing Page for Registration', () => {
+  cy.selectRegistration('Published registration with DOI', published);
+  cy.get('[data-testid=public-registration-status]').should('be.visible');
+});
+And('the Registration has a DOI Link', () => {
+  cy.get('[data-testid=public-registration-doi-link]').should('be.visible');
+});
+
+And('the Request DOI item is marked as Approved in their Worklist', () => {
+  cy.get('[data-testid="menu-button"]').click();
+  cy.get('[data-testid="worklist-link"]').click();
+});
+
+//   @1244
+//   Scenario: A Curator declines a DOI request
+Given('that a Curator enters a decline comment on a DOI request', () => {});
+When('they click Save', () => {});
+Then('the DOI request is marked as "Declined"', () => {});
+And('the request in the User\'s Worklist is updated to "Declined"', () => {});
+And("the request is removed from the Curator's Worklist", () => {});
+And('they see their Worklist', () => {});

```

Comment:
vil denne testen merkes som grønn og OK nå, selv om det mangler mulighet for å legge inn kommentar?

Reply:
Skal fjerne @test taggen fra scenariet